index.html

```diff
@@ -16,59 +17,97 @@
 
 <body>
 
-  
-     
-  <div class="container home">
+  <div class="top">
+    
+      <h1>Fun Bus</h1>
+      <nav>
+        <a href="#">Home</a>
+        <a href="#">About Us</a>
+        <a href="#">Blog</a>
+        <a href="#">Contact</a>
+      </nav>
    
+  </div>
+  
+  <div class="lines">
+  </div>
+
+  <div class="container-home">
+
+
+
+    <img src="img\fun-bus.jpg" class="fbus" alt="Fun Bus">
```

Comment:
Good use of alt, but you want it to be a little more descriptive of the actual image.
```suggestion
    <img src="img\fun-bus.jpg" class="fbus" alt="image of a school bus">
```